src/main/scala/ch/epfl/bluebrain/nexus/kg/directives/ProjectDirectives.scala

```diff
@@ -59,7 +59,7 @@ object ProjectDirectives {
   }
 
   private def addDefaultMappings(project: Project) = {
-    val pm = project.prefixMappings + ("nxv" -> nxv.base) + ("nxs" -> Schemas.base) + ("nxc" -> Contexts.base) + ("resource" -> Schemas.resourceSchemaUri) + ("base" -> project.base)
+    val pm = project.prefixMappings + ("nxv" -> nxv.base) + ("nxs" -> Schemas.base) + ("nxc" -> Contexts.base) + ("resource" -> Schemas.resourceSchemaUri) + ("elastic" -> nxv.defaultElasticIndex.value) + ("elastic" -> nxv.defaultElasticIndex.value)
```

Comment:
duplicate definition

Reply:
as per @samuel-kerrien's suggestion we should rename the path from `elastic` to `elasticsearch`